maquetado/login.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Log In</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+    <form class="info-container column">
+      <img src ="./assets/wolox.png" class="login-img" alt="Wolox">
+      <label for="email" class="input-label">Email</label>
+      <input name="email" id="email" class="input-field" type="email">
+      <label for="password" class="input-label">Password</label>
+      <input name="password" id="password" class="input-field" type="password">
+      <a type="submit" class="submit-button">Log in</a>
+      <a type="submit" class="redirect-button">Sign Up</a>
```

Comment:
This should have a `ref` attribute pointing to the sing-up page. Since in this PR you still don't have it, you can just use a placeholder like so 

```suggestion
      <a href="#" class="redirect-button">Sign Up</a>
```